.github/workflows/canary-integration-test.yml

```diff
@@ -763,7 +763,6 @@ jobs:
         yq write -i tests/manifests/test-cluster-on-pvc-encrypted.yaml "spec.storage.storageClassDeviceSets[0].count" 2
         yq write -i tests/manifests/test-cluster-on-pvc-encrypted.yaml "spec.storage.storageClassDeviceSets[0].volumeClaimTemplates[0].spec.resources.requests.storage" 6Gi
         kubectl create -f tests/manifests/test-cluster-on-pvc-encrypted.yaml
-        kubectl create -f cluster/examples/kubernetes/ceph/object-test.yaml
```

Comment:
Is this necessary to remove? IMO, it is always good to install *some* kind of storage in tests because a Ceph cluster with no pools is not going to happen in the real world, and often adding pools can change how Ceph behaves.